app/models/manageiq/consumption/showback_event.rb

```diff
@@ -45,7 +45,7 @@ def generate_data
       next unless resource.type.ends_with?(measure_type.category)
       self.data[measure_type.measure] = {}
       measure_type.dimensions.each do |dim|
-        self.data[measure_type.measure][dim] = 0
+        self.data[measure_type.measure][dim] = 0 unless measure_type.measure == "TEMPLATE"
```

Comment:
This creates the necessity to test in every function that the template is different
[ [ value ] ]?
Normally only one would be needed.